### ISSUE TITLE
Leads created with the IP Address of the server instead of the client

### ISSUE THREAD

**Issue by alejofernandez** [Marketo/Form-Proxy#1]
Hi!

I know this project is a bit old, but I'm trying to implement this in my marketo setup and I'm facing the following problem.
When I use this proxy the leads are created with the IP Address and location of my server (which runs in AWS EU)
<img width="344" alt="screen shot 2017-03-23 at 5 01 03 pm" src="https://cloud.githubusercontent.com/assets/1288192/24267874/70b13b9c-0fea-11e7-8d7c-f597940abc75.png">
But when I use the webform endpoint the leads are created with the correct IP Address and location (in this case in Argentina, where I live).
<img width="332" alt="screen shot 2017-03-23 at 5 03 12 pm" src="https://cloud.githubusercontent.com/assets/1288192/24267970/ca589ece-0fea-11e7-9436-b976f1323ccf.png">

I tried adding the `X-Forwarded-For` header with the original ip address to the `curl` request, but it seems that the `/index.php/leadCapture/save` endpoint is not taking it into account.

Is there any workaround?

**Comment by nathanbrauer** [comment]
That's correct.  Unfortunately, this is a side effect of proxy submissions.
